src/docs/reference/modules/Security/README.md

```diff
@@ -22,7 +22,7 @@ The following configuration values can be customized:
 ```json
     "OrchardCore_Security": {
       "ContentSecurityPolicy": {},
-      "PermissionsPolicy": { "fullscreen": "self" },
+      "PermissionsPolicy": { fullscreen: "self" },
```

Comment:
Why remove the quotes, it looked correct to me

Reply:
Fixed